File: decompiled/sources/com/hjq/http/request/GetRequest.java
package com.hjq.http.request;

import androidx.lifecycle.LifecycleOwner;
import com.hjq.http.model.HttpMethod;

public final class GetRequest extends UrlRequest<GetRequest> {
    public GetRequest(LifecycleOwner lifecycleOwner) {
        super(lifecycleOwner);
    }

    /* access modifiers changed from: protected */
    public String getRequestMethod() {
        return HttpMethod.GET.toString();
    }
}

File: decompiled/sources/com/hjq/http/request/GetRequest.kt
package com.hjq.http.request

import androidx.lifecycle.LifecycleOwner
import com.hjq.http.model.HttpMethod

class GetRequest(
    lifecycleOwner: LifecycleOwner
) : UrlRequest<GetRequest>(lifecycleOwner) {
    override fun getRequestMethod(): String {
        return HttpMethod.GET.toString()
    }
}
